### DISCUSSION TITLE
Is sphinx-autobuild going to get a new release?

### DISCUSSION THREAD

**Discussion by jdillard** [executablebooks/meta#934] [General]
It looks like there was some recent commit activity, but no new release. I am interested in getting the recent https://github.com/executablebooks/sphinx-autobuild/pull/109 into a release as I think it will fix an issue I have with turning warnings into errors.

**Comment by choldgraf** [comment]
I believe that @pradyunsg and @GaretJax are the two people with PyPI push access, any chance that either of you could make a release or add another executable books maintainer to the PyPI list so that we can set up a "release via GitHub Actions" workflow like exists in the other repositories here?

BTW, it would be nice to merge this one before a release if y'all think it'd be useful :-)

- https://github.com/executablebooks/sphinx-autobuild/pull/123

**Reply by jdillard** [reply]
Looks like progress is being tracked here: https://github.com/executablebooks/sphinx-autobuild/issues/126

Just adding that for any other wary travelers that come across this. Thanks for moving this along!

**Comment by jdillard** [comment]
Just checking in on this in case it slipped through the cracks. Looks like progress was made in those threads, but not sure where it ended up.

**Reply by choldgraf** [reply]
Mostly there is limited time to go through the process of a release. I'm happy to provide merge rights if others want to make releases and help maintain. I don't have time to do it myself though.

**Reply by jdillard** [reply]
@choldgraf sorry for the late reply, but I didn't want to accept merge rights on https://github.com/executablebooks/sphinx-autobuild until I felt I had the bandwidth to actually help, and I feel I have the time to contribute with releases/maintenance now if you are still interested. Thanks!
